buildenv/jenkins/JenkinsfileBase

```diff
@@ -541,9 +541,14 @@ def testBuild() {
 			if (!params.KEEP_WORKSPACE) {
 				// cleanWs() does not work in some cases, so set opts below
 				cleanWs notFailBuild: true, disableDeferredWipeout: true, deleteDirs: true
-			}
+				// clean up remaining core files in tmp folder
+				try {
+					sh "find /tmp/ -name '*core*' -print 2>/dev/null -exec rm -f {} \\;"
+				} catch (Exception ex) {
+					echo "cleaned up part of the core files."
```

Comment:
I do not think this echo message is informative. If we do not have permission to delete the file, do nothing. Unless we can print the file name, otherwise, I would prefer not to have the echo message.